### PR TITLE
navigation bar

### PR DESCRIPTION
**Pull request by abhishektanwar**
Files to ignore:
- All CSS files (already checked during HTML phase)

Files to check:
- components/Header/index.jsx

[PREVIEW LINK] : https://trekshark.netlify.app/

### CODE REVIEW

**Review by Muzammil-khan710 — APPROVED** [review]
Small PR..
Code is clean
 Variable Naming are upto the mark
It's good to merge..

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <Logo />
+      <SearchBar />
+      <div className="nav-section">
+        <BadgeIconButton
+          badgeIconButtonWrapper={"badge-icon-button-wrapper"}
+          badgeNumber={2}
+          icon={"far fa-heart"}
+          onClick={() => {
+            console.log("a");
+          }}
```
Comment (dev-prabhat) [inline src/components/Header/index.jsx]: delete console codes before pushing to github

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Nav Component looks clean, code is readable but delete some code like console.log on Event Listener

File: src/components/Header/Header.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+
+/* badge icon button */
+.badge-status-number{
+  top: -2.2rem;
+}
+
+.badge-icon i{
+  font-size: 3.4rem;
+  /* color: #D1D1D1; */
```
Comment (itsmebipul00) [inline src/components/Header/Header.css]: Comments could be removed

**Review by itsmebipul00 — APPROVED** [review]
Clean code, nice nav ui, good to merge

**Review by zoef-yt — APPROVED** [review]
The code looks good. Remove the commented code. 
The only suggestion I have is the icon size is a little big. The navbar looks great. Good to merge. Good work.

File: src/components/Header/index.jsx (hunk 1 above)
Comment (abhishektanwar) [reply src/components/Header/index.jsx]: fixed

File: src/components/Header/Header.css (hunk 2 above)
Comment (abhishektanwar) [reply src/components/Header/Header.css]: fixed

**Pull request merged**